src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
```

Comment:
Well defined class, perhaps you can consider defining as abstract class instead, for extensibility reasons in the future. You can consider defining the "crossstick" String that were found and separately defined in Event, Deadlines, Todo classes respectively.